lib/sensors/include/UltraSonicSensor.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Pwmln.h"
+#include "mbed.h"
```

Comment:
i believe we don't need mbed.h here. we've set up cmake so it automatically gets added when you link against the mbedos target.